### PR TITLE
Create user balance page

### PR DESCRIPTION
**Pull request by makspotter**

### CODE REVIEW

File: src/app/data/services/http/http.service.ts [hunk 1]
```diff
@@ -109,6 +113,50 @@ export class HttpService {
   getMaticPrices(): Observable<{ usd: number }> {
       return this.http.get<{ usd: number }>(`${this.localApi}/matic-info`);
   }
+  
+  getEthereumPools(): Observable<Pool[]> {
```
Comment (belbix) [inline src/app/data/services/http/http.service.ts:117]: http.service is basic implementation for http interractions
specific logic better to move to dedicated files

File: src/app/data/static/vault-names.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
+export const vaultNames = {
```
Comment (belbix) [inline src/app/data/static/vault-names.ts:1]: need to use names from harvest API

File: src/environments/environment.prod.ts [hunk 3]
```diff
@@ -6,4 +6,9 @@
 
 export const environment = {
   production: true,
+  APP_ETH_PARSER_URL: 'https://ethparser.herokuapp.com',
```
Comment (belbix) [inline src/environments/environment.prod.ts:9]: need to use sttings, this config will not work on heorku

File: src/app/data/services/http/http.service.ts (hunk 1 above)
Comment (makspotter) [reply src/app/data/services/http/http.service.ts]: fixed

File: src/environments/environment.prod.ts (hunk 3 above)
Comment (makspotter) [reply src/environments/environment.prod.ts]: fixed

**Comment by EmreKspToken** [comment]
EmreKspToken

**Pull request merged**